releasetool/commands/tag/nodejs.py

```diff
@@ -133,14 +133,16 @@ def create_release(ctx: TagContext) -> None:
             ]
         )
     else:
-        # TODO(sofisl): move the non-manifest release to release-please too
-        # for consistency:
-        ctx.github_release = ctx.github.create_release(
-            repository=ctx.upstream_repo,
-            tag_name=ctx.release_version,
-            target_commitish=ctx.release_pr["merge_commit_sha"],
-            name=f"{ctx.release_version}",
-            body=ctx.release_notes,
+        subprocess.check_output(
+            [
+                "npx",
+                "release-please",
+                "github-release",
+                f"--package-name={ctx.release_version}"
```

Comment:
you can just get rid of `--package-name`, `releaae-please` will automatically grab it from `package.json`.